crates/router/src/compatibility/wrap.rs

```diff
@@ -30,7 +30,7 @@ where
     error_stack::Report<E>: services::EmbedError,
     errors::ApiErrorResponse: ErrorSwitch<E>,
     T: std::fmt::Debug,
-    A: AppStateInfo,
+    A: AppStateInfo + Clone,
```

Comment:
clone probably not needed

Reply:
We need it or else we would get a compilation error.